ztp/gitops-subscriptions/hack/patch-argocd-dev.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# For dev use; if you will add the kustomize plugin binaries in the same repo->directory where the sitconfig.yaml exist
+# run this script to add configManagementPlugins to the ../argocd/deployment/argocd-openshift-gitops-patch.json patch
+# then re-patch the ArgoCD->openshift-gitops instance with the changes. Finally uncomment the ../argocd/deployment/clusters-app.yaml
+# plugin field and apply changes.
```

Comment:
Maybe this could patch both?

Reply:
I think no need to make patch for the App. Usually If some one will use this hack, he will need to have his own clusters-app.yaml (with another name) not using the one shared with others.

Reply:
done